nodes/surface/revolution_surface.py

```diff
@@ -6,31 +6,43 @@
 from bpy.props import FloatProperty, EnumProperty, BoolProperty, IntProperty
 
 from sverchok.node_tree import SverchCustomTreeNode
+from sverchok.utils.sv_transform_helper import AngleUnits, SvAngleHelper
 from sverchok.data_structure import updateNode, zip_long_repeat, ensure_nesting_level
 from sverchok.utils.curve import SvCurve
 from sverchok.utils.surface.algorithms import SvRevolutionSurface
 
 
-class SvRevolutionSurfaceNode(SverchCustomTreeNode, bpy.types.Node):
```

Comment:
`SvAngleHelper` should be first in the row. Python takes rightmost class first than it takes next left class and overrides methods and so on.